### PR TITLE
Add pipe for changing gender color

### PR DESCRIPTION
**Pull request by dhutaryan**

### CODE REVIEW

File: src/app/heroes/hero/hero.component.css [hunk 1]
```diff
@@ -0,0 +1,11 @@
+.text-primary {
+    color: #007bff;
```
Comment (dbl7) [inline src/app/heroes/hero/hero.component.css:2]: Create separate file with colors

File: src/app/heroes/hero/gender-class.pipe.ts [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    }
+
+    if (gender === 'Female') {
+      return 'text-danger';
+    }
+
+    if (gender === 'Not available') {
+      return 'text-muted';
+    }
+  }
```
Comment (dbl7) [inline src/app/heroes/hero/gender-class.pipe.ts]: How do you think, what's wrong with this implementation?

**Review by dbl7 — CHANGES_REQUESTED** [review]

File: src/app/heroes/hero/hero.component.css (hunk 1 above)
Comment (dhutaryan) [reply src/app/heroes/hero/hero.component.css:2]: Won't fix due to by using css

**Review by dbl7 — APPROVED** [review]

File: src/app/heroes/hero/gender-class.pipe.ts (hunk 2 above)
Comment (dhutaryan) [reply src/app/heroes/hero/gender-class.pipe.ts]: Fixed

**Pull request merged**